### PR TITLE
Prep. for potential 1.2.0 release

### PR DESCRIPTION
**Pull request by mrutkows**

### CODE REVIEW

**Comment by dgrove-oss** [comment]
@mrutkows -- be sure to update the end year in NOTICE.txt too...first release of 2021 for this component.

**Comment by mrutkows** [comment]
> @mrutkows -- be sure to update the end year in NOTICE.txt too...first release of 2021 for this component.

Will do Dave.  Just hoping this first commit on the draft builds with latest hashes...

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,35 +19,46 @@
 
 # Changelog
 
+## v1.2.0
+
+- Update for travis migration (#492)
```
Comment (rabbah) [inline CHANGELOG.md:24]:
```suggestion
- Update for Travis CI migration (#492)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,35 +19,46 @@
...
+- bump openwhisk-client-go dependency (#493)
```
Comment (rabbah) [inline CHANGELOG.md:25]:
```suggestion
- Bump openwhisk-client-go dependency (#493)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,35 +19,46 @@
...
+- remove trailing slash on apihost #481 (#485)
```
Comment (rabbah) [inline CHANGELOG.md:26]:
```suggestion
- Remove trailing slash on APIHOST #481 (#485)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -19,35 +19,46 @@
...
+- recognize .rs extension as a Rust action kind (#495)
```
Comment (rabbah) [inline CHANGELOG.md:27]:
```suggestion
- Recognize .rs extension as a Rust action kind (#495)
```

**Comment by mrutkows** [comment]
@rabbah Thanks Rodric, the lowercase was direct cut/paste from the actual commit messages and was debating if I should uppercase them for visual consistency.

**Review by dgrove-oss — APPROVED** [review]
LGTM

**Pull request merged**